swupd/bundleinfo.go

```diff
@@ -94,7 +94,12 @@ func (m *Manifest) addFilesFromBundleInfo(c config, version uint32) error {
 	for fpath := range m.BundleInfo.Files {
 		fullPath := filepath.Join(chrootDir, fpath)
 		fi, err := os.Lstat(fullPath)
+		if os.IsNotExist(err) {
+			fmt.Fprintf(os.Stderr, "Warning: Missing file, assuming %%ghost: %s\n", fpath)
```

Comment:
Is this safe for a quick hack? The correct solution should filter out the %ghost files before they are added to the bundle's file list.
If this doesn't break anything, I'm ok with it, but it would be nice to also have a TODO comment explaining what the proper fix should be.

Reply:
This is reasonably safe. The only unsafe path would be if people manually muck around the fullimage, but that is already out of scope - nothing protects against that.

We already catch all dnf errors anyway.

Reply:
Alright, then lets do it!